README.md

```diff
@@ -197,16 +197,106 @@ yarn install --force
 yarn reassure
 ```
 
-### CI integration
+## CI setup
+
+To make setting up the CI integration and all prerequisites more convenient, we have prepared a CLI command which will generate all necessary templates for you to get started with.
+
+Simply run:
+
+```bash
+yarn reassure init
+```
+
+This will generate the following file structure
+
+```
+├── <ROOT>
+│   ├── .reassure/
+│   ├──  reassure-tests.sh
+│   ├── dangerfile.ts (conditional)
+│   ├── dangerfile.reassure.ts (conditional)
+│   └── .gitignore (conditional)
+```
+
+### Options
+
+You can also use the following options in order to further adjust the script
+
+#### `--verbose` (optional)
+
+This is one of the options controlling the level of logs printed into the command prompt while running reassure scripts. It will
+
+#### `--silent` (optional)
+
+Just like the previous, this option also controls the level of logs. It will suppress all logs besides explicit errors.
+
+#### `--no-ascii-art` (optional)
+
+Just like the previous, this option also controls the level of logs. It will suppress ascii art based Hello and Bye messages.
+
+#### `--javascript` (optional)
+
+By default Reassure scripts will generate TypeScript files. You can use this option, if you'd like to generate JavaScript files instead.
+
+### Scaffolding
+
+#### `dangerfile.ts and dangerfile.reassure.ts`
+
+If your project already contains a `dangerfile`, the CLI will not override it in any way. Instead, it will generate a `dangerfile.reassure.ts` file which will allow you to compare and update your own at your own convenience.
+
+#### `.gitignore`
+
+If .gitignore file is present and no mentions of `reassure` appear within it, the script will append the `.reassure/` directory to its end.
+
+#### `reassure-tests.sh`
```

Comment:
Looks like this file will be generated by Reassure init. We could keep this section brief, mention the purpupose of that file and that it will be generated by Reassure init, while having more details in the github docs. @Xiltyn wdyt?